afkak/consumer.py

```diff
@@ -128,7 +132,8 @@ def __init__(self, client, topic, partition, processor,
                  max_buffer_size=None,
                  request_retry_init_delay=REQUEST_RETRY_MIN_DELAY,
                  request_retry_max_delay=REQUEST_RETRY_MAX_DELAY,
-                 request_retry_max_attempts=0):
+                 request_retry_max_attempts=0,
+                 auto_offset_reset='earliest'):
```

Comment:
This should default to 'fail', since having the wrong offset as a consumer is a serious, data-losing error.